File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples11.cs
// <snippetlinqexamples11>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var list_join = (from a in svcContext.AccountSet
                   join c in svcContext.ContactSet
                   on a.PrimaryContactId.Id equals c.ContactId
                   where a.Name == "Contoso Ltd" &&
                   a.Address1_Name == "Contoso Pharmaceuticals"
                   select a).ToList();
  foreach (var c in list_join)
  {
   System.Console.WriteLine("Account " + list_join[0].Name
       + " and it's primary contact "
       + list_join[0].PrimaryContactId.Id);
  }
 }

// </snippetlinqexamples11>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples11.vb
' <snippetlinqexamples11>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim list_join = ( _
      From a In svcContext.AccountSet _
      Join c In svcContext.ContactSet _
      On a.PrimaryContactId.Id Equals c.ContactId _
      Where a.Name.Equals("Contoso Ltd") _
      And a.Address1_Name.Equals("Contoso Pharmaceuticals") _
      Select a).ToList()
  For Each c In list_join
   Console.WriteLine("Account " &amp; list_join(0).Name _
                     &amp; " and it's primary contact " _
                     &amp; list_join(0).PrimaryContactId.Id.ToString())
  Next c
 End Using

' </snippetlinqexamples11>